test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -8728,6 +8728,28 @@ def forward(self, x, y):
         y = torch.full_like(x, True)
         self.run_test(MinimumModel(), (x, y))
 
+    @skipIfUnsupportedMinOpsetVersion(12)
+    def test_maximum_dtypes(self):
+        class MinimumModel(torch.nn.Module):
```

Comment:
I witness an event of copy-and-pasting. lol
```suggestion
        class MaximumModel(torch.nn.Module):
```

Reply:
:satisfied: